From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ram Muthiah <rammuthiah@google.com>
Date: Tue, 28 Jan 2020 14:10:57 -0800
Subject: ANDROID: gki_defconfig: Set IKHEADERS back to =y

This reverts I3624fa4eb40a8aa726275027aa2b2d5bd635ceda
The build time vintf check has been updated to expect IKHEADERS=y.

Bug: 139431025
Bug: 143488798
Test: TreeHugger
Change-Id: I60ede1c1fde3b8e011450a94d467f6c393613a7a
Signed-off-by: Ram Muthiah <rammuthiah@google.com>
---
 arch/arm64/configs/gki_defconfig | 2 +-
 arch/x86/configs/gki_defconfig   | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -9,7 +9,7 @@ CONFIG_TASK_IO_ACCOUNTING=y
 CONFIG_PSI=y
 CONFIG_IKCONFIG=y
 CONFIG_IKCONFIG_PROC=y
-CONFIG_IKHEADERS=m
+CONFIG_IKHEADERS=y
 CONFIG_UCLAMP_TASK=y
 CONFIG_MEMCG=y
 CONFIG_MEMCG_SWAP=y
diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -10,7 +10,7 @@ CONFIG_TASK_IO_ACCOUNTING=y
 CONFIG_PSI=y
 CONFIG_IKCONFIG=y
 CONFIG_IKCONFIG_PROC=y
-CONFIG_IKHEADERS=m
+CONFIG_IKHEADERS=y
 CONFIG_UCLAMP_TASK=y
 CONFIG_MEMCG=y
 CONFIG_MEMCG_SWAP=y
